docs/compute-engine/quickstart_android.md

```diff
@@ -1,19 +1,172 @@
+which you can use in your own Android app to perform inference with the [TensorFlow Lite Java inference APIs](https://www.tensorflow.org/lite/guide/inference#load_and_run_a_model_in_java).
+In this guide, we use the [TensorFlow Lite Android image classification app](https://github.com/tensorflow/examples/tree/master/lite/examples/image_classification/android)
+as an example.
+We provide a bash script to build a custom TensorFlow Lite AAR. The AAR package
+created with LCE build script, in addition to the built-in TensorFlow Lite operators,
+includes custom LCE operators which are registered by the TensorFlow Lite interpreter.
+
+In the following sections, we describe how to build a LCE-compatible
+TensorFlow Lite AAR package and use it in TensorFlow Lite Android image
+classification app.
+We rely on TensorFlow Lite Java inference APIs to perform inference with a Larq
+model. To get started with TensorFlow Lite on Android, we recommend reading the
+Tensorflow Lite [Android quickstart](https://www.tensorflow.org/lite/guide/android)
+carefully before proceeding with the next steps in this guide.
+
+### 1. Build in Android Studio ###
+
+To download and build the TensorFlow Lite Android image classification app
+in [Android Studio](https://developer.android.com/studio), follow the instructions
+[here](https://github.com/tensorflow/examples/blob/master/lite/examples/image_classification/android/README.md).
+For an explanation of the source code,
+see [here](https://github.com/tensorflow/examples/blob/master/lite/examples/image_classification/android/EXPLORE_THE_CODE.md).
+
+### 2. Add LCE-compatible TensorFlow Lite AAR to the project ###
+
+#### Use the LCE AAR from Bintray ####
+To use LCE AAR in the android app, we recommend using the
+[LCE package hosted at Bintray](https://bintray.com/plumeraihq/larq-compute-engine).
+Modify the `build.gradle` dependencies in the anroid project to include the
+`lce-lite` target:
+
+```
+allprojects {
+    repositories {
+        maven {
+            url  "https://dl.bintray.com/plumeraihq/larq-compute-engine"
+        }
+    }
+}
+dependencies {
+    implementation 'org.larq:lce-lite:0.1.2
+}
+```
+
+This AAR includes binaries for 'arm64-v8a', 'x86_64', 'x86' [Android ABIs](https://developer.android.com/ndk/guides/abis).
+Please note that currently hand-optimized LCE operators are available only for 'arm64-v8a'.
+To include binaries only for Android ABIs you need to support, modify `abiFilters`
+in the Gradle build as described [here](https://www.tensorflow.org/lite/guide/android#use_the_tensorflow_lite_aar_from_jcenter).
+
+!!! note
+    In TensorFlow Lite image classification app, you need to also remove
+    the standard TensorFlow Lite dependency by commenting out the following
+    line in `build.gradle` file:
+
+    ```
+    dependencies {
+        // Comment out the following line
+        // implementation 'org.tensorflow:tensorflow-lite:0.0.0-nightly'
+    }
+    ```
```

Comment:
```suggestion
    }
    ```
    Note that the dependencies `ensorflow-lite-gpu` and `tensorflow-lite-support` are still required!
```